TF_Service_dotNet/TouchFree_Service/Program.cs

```diff
@@ -7,6 +11,14 @@ public class Program
     {
         static void Main(string[] args)
         {
+#if !DEBUG
+            FileStream filestream = new FileStream("log.txt", FileMode.Create);
```

Comment:
Not sure how feasbile this is when booting the app but it would be nice for this to go into the configured TF Config directory rather than alongside the executable. Not a hard requirement for approval though.